### ISSUE TITLE
Is possible working on ESP32 S3?

### ISSUE THREAD

**Issue by b4k0s** [justcallmekoko/ESP32Marauder#416]
Is this possible?

**Comment by mirkokral** [comment]
When custom compiling, it IS possible. I use my ESP32-S3-DEV-KIT-N8R8 on a perfboard with a display and all works fine.

**Comment by b4k0s** [comment]
Ahhh ok ok! i have a LilyGo Esp32-s3 and i wanted to try Marauder on It. You think about compiling for that board? thanks for your answer!

**Comment by XANKXANK** [comment]
i have seeed esp32c3 and it works fine with marauder

**Comment by alisdairg** [comment]
> ... the Esp32-s3 Just dont Power UP, keeps doing that sound connected/disconnected device on windows.

That sounds more like a cheap USB cable not providing enough power. I've seen that loads of times.

**Comment by b4k0s** [comment]
> > ... the Esp32-s3 Just dont Power UP, keeps doing that sound connected/disconnected device on windows.
> 
> That sounds more like a cheap USB cable not providing enough power. I've seen that loads of times. 
> 

Its a original Xiaomi cable. Btw my Esp32-s3 Works Fine running NerdMiner V2 (and i have a battery). So i think its not the cable.![IMG_20231202_115548.jpg](https://github.com/justcallmekoko/ESP32Marauder/assets/152666652/89959db4-a6f5-4e18-a35f-dc54d7eaebdc)

**Comment by b4k0s** [comment]
any help?

**Comment by nysot94** [comment]
> ![IMG-20231203-WA0022.jpeg](https://github.com/justcallmekoko/ESP32Marauder/assets/152666652/c4f202d5-cd9f-4e91-ac95-2f8aa092b99c)
> 
> 

How you do that?

**Comment by b4k0s** [comment]
> > ![IMG-20231203-WA0022.jpeg](https://private-user-images.githubusercontent.com/152666652/287542030-c4f202d5-cd9f-4e91-ac95-2f8aa092b99c.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI1NzMxOTAsIm5iZiI6MTcwMjU3Mjg5MCwicGF0aCI6Ii8xNTI2NjY2NTIvMjg3NTQyMDMwLWM0ZjIwMmQ1LWNkOWYtNGU5MS1hYzk1LTJmOGFhMDkyYjk5Yy5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMTRUMTY1NDUwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmE4ODMwODU0ZmZlMWVjMGE4NDk5ZWUwNGUxNGUzY2ZhMDUzN2I1NDU1M2QxOTQ3YmIwMjk3Y2MxYTAzZWZjNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.KywfLTVvLXwR8cITouTxGq5DPTdj9KiCgekcW3g1KjY)
> 
> How you do that?

i used the Dirty Flasher.

**Comment by nysot94** [comment]
I got my Lilygo s3 today, I try but don't work

**Comment by nysot94** [comment]
There is some YouTube tutorial? I'm noob...

**Comment by gousdail** [comment]
have you managed to get marauder to work on your LilyGo Esp32-s3 ?

**Comment by b4k0s** [comment]
> have you managed to get marauder to work on your LilyGo Esp32-s3 ?

yes, but its just a standalone... needs to connect via terminal to use commands on it.

**Comment by avipars** [comment]
Can you share the bin file?

**Comment by avipars** [comment]
> When custom compiling, it IS possible. I use my ESP32-S3-DEV-KIT-N8R8 on a perfboard with a display and all works fine.

@mirkokral

**Comment by mirkokral** [comment]
> > When custom compiling, it IS possible. I use my ESP32-S3-DEV-KIT-N8R8 on a perfboard with a display and all works fine.
> 
> [@mirkokral](https://github.com/mirkokral)

 I am sorry, but I no longer have the bin file. It was possibly deleted during my home folder cleanup.
